test/suite/functional/Generator/Request/PatchResourceRequest74.php

```diff
@@ -25,10 +25,13 @@ class PatchResourceRequest implements RequestInterface
 
     private string $contentType = 'application/json';
 
-    public function __construct(PatchResourceRequestBody $patchResourceRequestBody, string $accept = 'application/json')
+    private string $apiKey;
+
+    public function __construct(PatchResourceRequestBody $patchResourceRequestBody, string $apiKey, string $accept = 'application/json')
```

Comment:
here you don't use  Constructor property promotion like here: https://github.com/DoclerLabs/api-client-generator/pull/104/files#diff-b34e02907a5e05b7f92a31e35eca839f64e09d58f408020b1cc8c73d79a4ab21R19

Reply:
true, because this file is used in the test, to compare with file created by generator using php 7.4 😅

Reply:
I should have figure it out from the name 😅